frontend/rolecall/src/styles/_colors.scss

```diff
@@ -6,3 +6,4 @@ $gray: #6d6d6d;
 $lightGray: #9c9c9c;
 $lightLightGray: #b1b1b1;
 $black: black;
+$AAOrangeBorder: #ff6666;
```

Comment:
Let's call the variable `$AAOrangeColor` because I would expect something with a suffix of `Border` to be defining a border style like `1px solid #f3f3f3`, not just a color. Also let's move it up next to `$AAOrange` since they are closely related.

It would even be great to rename `$AAOrange` to `$AAOrangeGradient` to avoid the naming confusion in the future.

Reply:
I made the changes you suggested.